assets/js/hooks/_sticky_actions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Hook to determine if a table row is horizontally scrollable (=> actions are sticky).
+ *
+ * Adds a 'stuck' attribute to the sticky element which can then be used for styling.
+ */
+export default {
+  mounted() {
+    const sticky = this.el.querySelector('.sticky')
+
+    this.observer = new IntersectionObserver(
+      ([entry]) => { sticky.toggleAttribute('stuck', entry.intersectionRatio < 1)},
```

Comment:
I just found another bug... the `stuck` attribute is removed on re-renders (select item or search for example). I'm probably going crazy here... 🥴